src/chrome/content/rules/Tangerine-Bank.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+	Invalid certificates:
+		- forwardthinking.tangerine.ca
+		- e.tangerine.ca
+		- links.e.tangerine.ca
+
+	Timeout:
+		- tangerine.com
+		- www.tangerine.com
+		- brightwayforward.ca
+		- www.brightwayforward.ca
+-->
+
+<ruleset name="Tangerine Bank">
+	<target host="tangerine.ca" />
+	<target host="www.tangerine.ca" />
+	<target host="secure.tangerine.ca" />
+	<target host="info.tangerine.ca" />
+	<target host="api.tangerine.ca" />
+	<target host="upload.tangerine.ca" />
+	<target host="readonly.tangerine.ca" />
+	<target host="m.tangerine.ca" />
+	<target host="ofx.tangerine.ca" />
+	<target host="tangerine.com" />
+	<target host="www.tangerine.com" />
+
+	<test url="https://tangerine.com/" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^https?://(www\.)?tangerine\.com/" to="https://$1tangerine.ca/" />
```

Comment:
I initally added that rule because `tangerine.com` times out over https. But it's probably not needed, so I'm changing it to an http-to-https redirect.